netmaster/master/netmaster_test.go

```diff
@@ -296,3 +308,33 @@ func TestVxlanConfigWithLateHostBindings(t *testing.T) {
 
 	verifyKeys(t, keys)
 }
+
+// Tests for https://github.com/contiv/netplugin/issues/214
+func TestVxlanConfigPktTagOutOfRange(t *testing.T) {
+	cfgBytes := []byte(`{
+    "Tenants" : [{
+        "Name"                  : "tenant-one",
+        "DefaultNetType"        : "vxlan",
+        "SubnetPool"            : "11.1.0.0/16",
+        "AllocSubnetLen"        : 24,
+        "Vxlans"                : "2001-3000",
+        "Networks"  : [{
+            "Name"              : "orange",
+            "PktTag"            : 1300
+        }]
+    }]}`)
+
+	initFakeStateDriver(t)
+	defer deinitFakeStateDriver()
+
+	// Create the Tenant and Network
+	applyConfig(t, cfgBytes)
+	fakeDriver.DumpState()
+
+	keys := []string{"tenant-one"}
+	verifyKeys(t, keys)
+
+	keys = []string{"nets/orange"}
+	fakeDriver.DumpState()
```

Comment:
will the above print a lot of gunk during test runs? Can we do that only when test case fails, or better yet remove it all together.